### PR TITLE
feat(reporter): add support for source map comments

### PR DESCRIPTION
**Pull request by glan**
- Allow an embedded sourcemap to be extracted from the source file.
- Add source map support to stack traces from sources built using browserify with debug enabled.

### CODE REVIEW

**Comment by glan** [comment]
Travis please test this

**Comment by glan** [comment]
@vojtajina I merged master back into the branch and Travis now builds fine. Is there anything else that needs doing?

**Comment by Dignifiedquire** [comment]
LGTM

File: lib/reporter.js [hunk 1]
```diff
@@ -41,6 +42,10 @@ var createErrorFormatter = function(basePath, emitter, SourceMapConsumer) {
 
       var file = findFile(path);
 
+      if (file && !file.sourceMap) {
```
Comment (maksimr) [inline lib/reporter.js]: Do not slow down this behavior preprocessing the files?

**Comment by Dignifiedquire** [comment]
@glan ping

**Comment by glan** [comment]
@maksimr @Dignifiedquire How about we put this feature behind a flag?

**Comment by maksimr** [comment]
@glan [this](https://github.com/demerzel3/karma-sourcemap-loader) plugin solve your problem?
Really I can not understand why karma should extract source map. Seems it's not karma responsibility.

Thanks

**Comment by Dignifiedquire** [comment]
@glan any thoughts about the comment from @maksimr ?

**Comment by dignifiedquire** [comment]
Sorry, but I'm closing this due to inactivity. Please feel free to reopen a new updated PR.

**Pull request closed without merging**